karpenter/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # CHANGELOG - karpenter
 
 <!-- towncrier release notes start -->
+## 1.3.0 / 2024-03-08
```

Comment:
Please undo the changes to this file, then install the latest version of `ddev` (lmk if you need help with that), then run:

```sh
ddev release changelog new added karpenter --message "New karpenter metrics."
```

You should see a file in the `changelog.d` directory, please commit that and push it.